src/components/styles/Theme.tsx

```diff
@@ -368,6 +381,17 @@ export const zopaTheme: AppTheme = {
   progressBar: {
     color: colors.brand,
   },
+  scrollableArea: {
+    scrollBarThumb: {
+      borderRadius: '4px',
+      border: `2px solid ${colors.brand}`,
+      bgColor: `${colors.brand}`,
+    },
+    scrollBarTrack: {
+      background: `${colors.greyLighter}`,
+      borderRadius: '4px',
+    },
+  },
   typography: {
     primary: `"Open Sans", Roboto, Helvetica, Arial, sans-serif`,
```

Comment:
do we need to theme font there too?

Reply:
ScrollableArea is just a container, the font styles are passed with the children I think